### ISSUE TITLE
error: the command "setlocal

### ISSUE THREAD

**Issue by PaxtonPenguin** [supertuxkart/stk-code#5095]
## Description

no i did not accidentally not include the other quote, that is the exact error message i got while building stk

## Steps to reproduce

1. open the .sln file
2. build the INSTALL thing
3. see it fail
## Configuration

STK release version: commit [97faf4a](https://github.com/supertuxkart/stk-code/commit/97faf4a492975646915c6d40bf81fe620eea2950)

STK Source (PPA, distribution package, official binary, etc.): git

## Additional information

Error message:
```
MSB3073: The command "setlocal
```

**Comment by Alayan-stk-2** [comment]
> 1. open the .sln file

No idea what that is.

Since the continuous integration manages to compile fine on Windows, I assume you probably missed some step in the instructions, or made a mistake.

The full instructions are here: https://github.com/supertuxkart/stk-code/blob/master/INSTALL.md

You can ask for help on the IRC here: https://web.libera.chat/?channels=#supertuxkart

Compiling for the first time can be tricky, but please don't open a new issue each time you get a problem building STK, not until you can be reasonably certain the problem is caused by a bug in the code.

**Comment by PaxtonPenguin** [comment]
.sln is the visual studio solution file

**Comment by PaxtonPenguin** [comment]
tried with msbuild, it worked, don't use visual studio
